src/routes/renderRoutes.js

```diff
@@ -1,41 +1,36 @@
 const db = require("../../sequelize/db");
 const es = require("../../database/elasticsearch/elastic");
 const pg = require("../../database/postgres/pg");
-
+const arangoModule = require("../../database/arango/arango");
+require("dotenv").config();
 const routes = {};
 
+// This is the 'host' url for a person's database credentials
+const dbHost = {
+  Postgres: "learndatabases.dev",
+  Elasticsearch: "elastic.learndatabases.dev",
+  Arango: process.env.ARANGO_URL,
+};
+
+const checkAccount = {
+  Postgres: ({ username }) => pg.userHasPgAccount(username),
+  Elasticsearch: (user) => es.checkAccount(user),
+  Arango: ({ username }) => arangoModule.checkIfDatabaseExists(username),
+};
```

Comment:
Shouldn't it be `process.env.HOST` for postgres/sequelize and `process.env.ES_HOST` for elastic?